src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -915,4 +923,44 @@ public void transientNotificationOnlyNodeWithItemTest() throws NoResponseExcepti
             pubSubManagerOne.deleteNode(nodename);
         }
     }
+
+    @SmackIntegrationTest
+    public void isEqualToOneTest() {
+        assertEquals(1,1);
+    }
+
+    @SmackIntegrationTest
+    public void sendMessageEventTest() throws NoResponseException, XMPPErrorException, NotConnectedException, InterruptedException, XmppStringprepException, TestNotPossibleException {
+        final String nodename = "sinttest-publish-item-nodename-" + testRunId;
+        final String needle = "test content " + Math.random();
+        LeafNode publisherNode = pubSubManagerOne.createNode(nodename);
+        try {
+            // Publish a new item.
+            publisherNode.publish( new PayloadItem<>( GeoLocation.builder().setDescription( needle ).build() ) );
+            Node subscriberNode = pubSubManagerTwo.getNode(nodename);    
+            ChatManager chatManager = ChatManager.getInstanceFor(connection);
+            final EntityBareJid subscriber = conTwo.getUser().asEntityBareJid();
+            subscriberNode.subscribe( subscriber );
+            Message m = new Message();
+            // Retrieve items and assert that the item that was just published is among them.
+            MessageEvent message = new MessageEvent();
+            message.setOffline(true);
+            message.setStanzaId(StanzaIdUtil.newStanzaId());
+            m.addExtension(message);
+            // final List<Item> items = publisherNode.getItems();
+            System.out.println(message.toString());
+            System.out.println(m.toString());
+
+            // EntityBareJid jid = JidCreate.entityBareFrom("itot@goodbytes.im");
+            Chat chat = chatManager.chatWith(subscriber);
+            chat.send(m);
+            // assertTrue( items.stream().anyMatch( stanza -> stanza.toXML( "" ).toString().contains("offline") ) );
```

Comment:
This is probably because you're still busy working on this code, but: every test should at least have one 'assertion'. If tests do not have assertions, they do not actually test anything!